translations/back_cover.tex/fr.po

```diff
@@ -89,7 +88,37 @@ msgstr ""
 "\\azure \\azure} \\\\ \\end{tabularx} }"
 
 #. type: Plain text
-#: sections/back_cover.tex:44
+#: sections/back_cover.tex:47
+#, fuzzy
+#| msgid ""
+#| "\\hommtable{14}{ \\centering \\textbf{\\pagetarget{Trade Table}{Trade "
```

Comment:
This doesn't look right, the French crew just fixed it.

Reply:
Good catch. That was a bad merge from `main` - fixed